en/editor/publish/build-guide.md

```diff
@@ -1,58 +1,81 @@
-# Build Process Introduction with FAQ
+# Introduction to the build process and FAQ guide
 
-The build process is mainly divided into two parts, the **General Build Process** and the **Platform Adaptation Process**. The adaptation processing logic for each platform will be embedded in the **Build** panel as a separate plugin. The build plugin system is then open and developers can dynamically embed some build parameters into the panel for use.
+## Introduction to Building Infrastructure
```

Comment:
Introduction to the infrastructure of the build?

Reply:
We would need to add more though. What I wrote makes sense, technically. We could also use:

Introduction to Building Infrastructure

Introduction to the infrastructure of building applications

The Infrastructure of building applications